auth-web/tests/unit/views/HomeView.spec.ts

```diff
@@ -0,0 +1,144 @@
+import { createLocalVue, mount } from '@vue/test-utils'
+import BcscPanel from '@/components/auth/BcscPanel.vue'
+import HomeViewDev from '@/views/auth/HomeViewDev.vue'
+import InfoStepper from '@/components/auth/stepper/InfoStepper.vue'
+import LoginBCSC from '@/components/auth/LoginBCSC.vue'
+import TestimonialQuotes from '@/components/auth/TestimonialQuotes.vue'
+import Vue from 'vue'
+import VueRouter from 'vue-router'
+import Vuetify from 'vuetify'
+import Vuex from 'vuex'
+
+Vue.use(Vuetify)
+Vue.use(VueRouter)
+const router = new VueRouter()
+const vuetify = new Vuetify({})
+
+// Prevent the warning "[Vuetify] Unable to locate target [data-app]"
+document.body.setAttribute('data-app', 'true')
+jest.mock('../../../src/plugins/i18n', () => {})
+
+describe('HomeView.vue', () => {
+  let wrapper: any
+  let userModule: any
+
+  beforeEach(() => {
+    const localVue = createLocalVue()
+    localVue.use(Vuex)
+
+    userModule = {
+      namespaced: true,
+      state: {
+        userProfile: {
+        }
+      },
+      actions: {
+        getUserProfile: jest.fn()
+      }
+    }
+
+    const orgModule = {
+      namespaced: true,
+      state: {
+        currentOrganization: {
+        }
+      },
+      actions: {
+        syncOrganizations: jest.fn(),
+        syncCurrentOrganization: jest.fn()
+      }
+    }
+
+    const store = new Vuex.Store({
+      state: {},
+      strict: false,
+      modules: {
+        user: userModule,
+        org: orgModule
+      }
+    })
+
+    wrapper = mount(HomeViewDev, {
+      store,
+      localVue,
+      router,
+      vuetify,
+      mocks: {
+        $t: (mock) => mock
+      }
+    })
+
+    jest.resetModules()
+    jest.clearAllMocks()
+  })
+
+  it('is a Vue instance', () => {
+    expect(wrapper.isVueInstance()).toBeTruthy()
+  })
+
+  it('renders the sub-components properly', () => {
+    expect(wrapper.find(InfoStepper).exists()).toBe(true)
+    expect(wrapper.find(TestimonialQuotes).exists()).toBe(true)
+    expect(wrapper.find(BcscPanel).exists()).toBe(true)
+  })
+
+  it('renders the correct buttons when authenticated', () => {
+    const bannerBtns = wrapper.vm.$el.querySelectorAll('.cta-btn')
+    const nameRequestBtn = bannerBtns[0]
+    const namedCompBtn = bannerBtns[1]
+    const numberedCompBtn = bannerBtns[2]
+    const manageBusinessBtn = bannerBtns[3]
+    const loginBtn = bannerBtns[4]
```

Comment:
Oh, I like this way of naming the buttons up front! Makes the 'expect' code much easier to read :)

Reply:
awesome, thanks!